docs/topics/mta-web-importing-an-app-list.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-importing-an-app-list_{context}"]
+= Importing an application list
+
+You can important a .csv file containing a list of applications and their attributes to the {ProductName} ({ProductShortName}) {WebName}.
+
+[NOTE]
+====
+Importing a list of applications only adds applications, it does not overwrite any existing ones.
+====
+
+.Procedure
+
+. Review the import file to ensure it contains all the required information in the required format.
+. In *Developer* view, click *Application Inventory*.
+. Click the Options menu {kebab} to the right of *Review*.
+. Click *Import*.
++
+// ![](/Tackle2/AddingApps/Import.png)
+. Browse to the desired file and click *Import*.
```

Comment:
click import was mentioned already in the previous line.

Reply:
Should probably be Open.

Reply:
@istein1 Please verify that the button you click when you browse to the file is named  "Open"